### DISCUSSION TITLE
Em que posso ajudar?

### DISCUSSION THREAD

**Discussion by dsmello** [BrasilAPI/BrasilAPI#237] [Q&A]
Bom dia, a todos,

Gostaria de saber em que posso ajudar?

Eu sei .:
-> Python
-> DevOps
-> Go

**Comment by DJDisclinc** [comment]
Go

**Comment by elirweb** [comment]
Boa tarde, a todos
Eu sei
C#,
Angular qualquer versão,
Node,
TypeScript,
Mongo DB e Sql Server,

**Comment by enieber** [comment]
@dsmello acredito que a principio o conhecimento de DevOps vai ser bem interessante para ajustar os pipelines ou até criar alguns pipelines.

**Comment by elirweb** [comment]
boa tarde,
no projeto, vi que tem uns pipelines em "Yarn", neste caso criaria em Yarn
com git actions?

Em seg., 23 de nov. de 2020 às 14:29, Enieber Cunha <
notifications@github.com> escreveu:

**Comment by enieber** [comment]
Atualmente só tem actions para testes e2e se quiserem implementar um pipeline de testes unitarios ou algum outro pipeline relacionado ao projeto talves esse projeto possa ajustar https://github.com/nektos/act

**Comment by elirweb** [comment]
para entender, com base neste " https://github.com/nektos/act" quer que eu
applica pipelines de teste unitários?

Em seg., 23 de nov. de 2020 às 14:45, Enieber Cunha <
notifications@github.com> escreveu:

**Comment by enieber** [comment]
@elirweb fique a vontade para implementar algum github actions, eu usei esse act para rodar o pipeline na minha maquina e assim saber se o pipeline está correto ou não

**Comment by elirweb** [comment]
Ok.

Em seg, 23 de nov de 2020 15:00, Enieber Cunha <notifications@github.com>
escreveu:
